rotkehlchen/tests/unit/globaldb/test_globaldb_cache.py

```diff
@@ -159,18 +188,37 @@ def test_velodrome_cache(optimism_inquirer):
     pools, gauges = read_velodrome_pools_and_gauges_from_cache()
     assert pools >= VELODROME_SOME_EXPECTED_POOLS
     assert gauges >= VELODROME_SOME_EXPECTED_GAUGES
-
-    with GlobalDBHandler().conn.read_ctx() as cursor:
-        addressbook_entries = DBAddressbook(optimism_inquirer.database).get_addressbook_entries(
-            cursor=cursor,
-            filter_query=AddressbookFilterQuery.make(),
-        )[0]
-        asset_identifiers = cursor.execute('SELECT identifier FROM assets').fetchall()
-
+    addressbook_entries, asset_identifiers = get_velodrome_addressbook_and_asset_identifiers(optimism_inquirer)  # noqa: E501
     assert all(entry in addressbook_entries for entry in VELODROME_SOME_EXPECTED_ADDRESBOOK_ENTRIES)  # noqa: E501
     assert all((identifier,) in asset_identifiers for identifier in VELODROME_SOME_EXPECTED_ASSETS)
 
 
+@pytest.mark.vcr()
+def test_convex_cache(ethereum_inquirer):
+    """Test convex pools querying and caching mechanism"""
+    convex_expected_pools = {  # some expected pools
+        string_to_evm_address('0x49Dd6BCf56ABBE00DbB816EF6664c4cf5bdd81A1'): 'dYFIETH-f',
+        string_to_evm_address('0xDe91Bf29ADF79FbfbbF0d646EAf024c0CB9fac25'): 'crvstUSDT-f',
+        string_to_evm_address('0x62f3C96017F2Ba9D83BD70500B738FEEebc5FFc6'): 'crvSTBT-f',
+        string_to_evm_address('0xFe4aC9cd3892BACbeA12C9185a577164f56831fD'): 'ETHxfrxETH-f',
+    }.items()
```

Comment:
Why did you use dict_items() here and below? Also most importantly you did not fix curve which I said you should.